tmt/utils.py

```diff
@@ -626,14 +628,39 @@ def _run(self,
         stdout_thread.start()
         if not join:
             stderr_thread.start()
+
+        # A bit of logging helpers for debugging duration behavior
+        start_timestamp = time.monotonic()
+
+        def _event_timestamp() -> str:
+            return f'{time.monotonic() - start_timestamp:.4}'
```

Comment:
Is the following timestamp format expected:

```
command event: 5.8e-06 running: set -eo pipefail; ./tmt-test-wrapper.sh </dev/null |& cat
```

Reply:
Removed in 3344a6d10c757f7a5e23714dbb5313034a2c8793.